### DISCUSSION TITLE
How should fields be grouped?

### DISCUSSION THREAD

**Discussion by mikeallbutt** [DiscipleTools/disciple-tools-theme#1155] [Q&A]
It is possible to move fields around and have them be in different tiles on the web (tabs/screens in the app). When you use the DT interface do you find that the fields are in the best logical location, order, sequence etc.

What would you do differently?

**Comment by ahillbilly** [comment]
This looks like a duplicate of https://github.com/DiscipleTools/disciple-tools-theme/discussions/1147

**Reply by mikeallbutt** [reply]
Yes I think you are right that it is asking the same type of question. I still hope that others will answer it.
